webapp-django/crashstats/manage/static/manage/js/symbols-uploads.js

```diff
@@ -0,0 +1,118 @@
+/*global alert:true location:true PaginationUtils:true */
+(function ($, document) {
+    'use strict';
+
+    function humanFileSize( bytes, precision ) {
+        var units = [
+           'bytes',
+           'Kb',
+           'Mb',
+           'Gb',
+           'Tb',
+           'Pb'
+        ];
+        if (isNaN( parseFloat(bytes)) || !isFinite(bytes)) {
+            return '?';
+        }
+        var unit = 0;
+        while (bytes >= 1024) {
+          bytes /= 1024;
+          unit++;
+        }
+        return bytes.toFixed(+precision) + ' ' + units[unit];
+    }
+
+    function start_loading() {
+        $('.pleasewait').show();
+    }
+
+    function stop_loading() {
+        $('.pleasewait').hide();
+    }
+
+    function fetch_uploads() {
+
+        start_loading();
+        var $form = $('#filter');
+        var url = $form.data('dataurl');
+        var data = $form.serialize();
+        $.getJSON(url, data, function(response) {
+            stop_loading();
+            $('.count b').text(response.count);
+            $('.count:hidden').show();
+            $('tbody tr', $form).remove();
+            var $tbody = $('tbody', $form);
+            $('tr', $tbody).remove();
+            function userURL(id) {
+                return response.user_edit_url.replace('0', id);
+            }
+            function contentURL(id) {
+                return response.symbols_content_url.replace('0', id);
+            }
```

Comment:
Yeah. You're right. 
I took that idea from another application where I make the JSON return EVERYTHING and then I do pagination in the webapp (angular). For that case I wanted to avoid repeating the URLs so much in the JSON and instead just sent a "URL template". 

It was premature optimization for this case. Fixed now.